runtime/compiler/runtime/RelocationRuntime.hpp

```diff
@@ -121,6 +121,83 @@ typedef struct TR_AOTRuntimeInfo {
 extern J9_CFUNC void
 printAOTHeaderProcessorFeatures(TR_AOTHeader * aotHeaderAddress, char * buff, const size_t BUFF_SIZE);
 
+enum TR_RelocationErrorCodeType
+   {
+   NO_ERROR   = 0x0,
+   MISC       = 0x1,
+   VALIDATION = 0x2,
+   RELOCATION = 0x4,
+   };
+
+#define RELO_ERRORCODE_SHIFT 4
+
+enum TR_RelocationErrorCode
```

Comment:
The reason for low tagging the enum values is so that if someone in the future adds a new error code, they don't have to go and update the value of all the error codes that follow it. Without the tagging, it also would not be possible to just add a new error code to the end because the compiler needs to store a SCC hint if the relocation failure was a validation failure.

Reply:
Added comment in [force push](https://github.com/eclipse-openj9/openj9/compare/eed5bda0d94753b6c86ff124756a972ac054411c..764e07f428212dcf6edff7f6f9244035a3a589d4).